Makefile

```diff
@@ -55,7 +55,7 @@ battletest: ## Run randomized, racing, code coveraged, tests
 
 e2etests: ## Run the e2e suite against your local cluster
 	go clean -testcache
-	CLUSTER_NAME=${CLUSTER_NAME} go test -p 1 -timeout 180m -v ./test/suites/... -run=${TEST_FILTER}
+	cd test && CLUSTER_NAME=${CLUSTER_NAME} go test -p 1 -timeout 180m -v ./suites/... -run=${TEST_FILTER}
```

Comment:
Do we want to execute in a subshell to avoid changing the directory? Not sure how this interacts with make.

Reply:
Make does execute this in a subshell by default